### PR TITLE
Feature/eslint config sonar update

### PR DESCRIPTION
**Pull request by penx**
update to eslint-config-sonar 0.0.7

(also fix minor linting issue raised by eslint-config-sonar@0.0.7)

### CODE REVIEW

**Comment by penx** [comment]
Deploy preview for *govuk-react* processing.

Building with commit 7720c397be2a1e3933ad55fcab55988876d85ec3

https://app.netlify.com/sites/govuk-react/deploys/5c925a194a55b50008a92f03

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "enzyme-to-json": "^3.3.4",
     "eslint": "^4.15.0",
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-sonar": "0.0.6",
+    "eslint-config-sonar": "0.0.7",
```
Comment (stevesims) [inline package.json]: wondering if we should put a hat on this

File: packages/constants/src/typography.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const TYPOGRAPHY_SCALE = {
     },
     tablet: {
       fontSize: 80,
-      lineHeight: 80 / 80,
+      lineHeight: 1, // 80 / 80
```
Comment (stevesims) [inline packages/constants/src/typography.js]: ah - yes :grin:

File: package.json (hunk 1 above)
Comment (penx) [reply package.json]: hat wouldn't make a difference at 0.0.x as every change is a breaking change

Comment (stevesims) [reply package.json]: ah - true.

perhaps that means `eslint-config-sonar` should get it's version updated to `0.1.0`?

**Pull request merged**